### PR TITLE
Feat/doc register

### PR DESCRIPTION
**Pull request by d1360cub**
Adds show password to login, adds modal to doctor register, adds doctor register

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -480,6 +480,18 @@
       "services": "Psicología clínica",
       "description": "Experiencia en tratamientos clínicos en adultos",
       "id": 6
+    },
+    {
+      "id": 7
```
Comment (mherreraatleantech) [inline db.json]: Por qué este objeto solo tiene el id?

File: src/Pages/DoctorSignup/DoctorSignup.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+function DoctorSignup() {
+  const [form, setForm] = useState({});
+  const handleChange = (event) => {
+    const { value, name } = event.target;
+    setForm({
+      ...form,
+      [name]: value,
+    });
+    console.log(form);
```
Comment (mherreraatleantech) [inline src/Pages/DoctorSignup/DoctorSignup.jsx]: No console.log

**Review by mherreraatleantech — CHANGES_REQUESTED** [review]

File: db.json (hunk 1 above)
Comment (d1360cub) [reply db.json]: deleted

File: src/Pages/DoctorSignup/DoctorSignup.jsx (hunk 2 above)
Comment (d1360cub) [reply src/Pages/DoctorSignup/DoctorSignup.jsx]: deleted

**Review by d1360cub — COMMENTED** [review]
modified

**Review by mherreraatleantech — APPROVED** [review]

**Review by mhbuiles — APPROVED** [review]

**Pull request merged**